scarlet2/initialization.py

```diff
@@ -23,9 +25,9 @@ def _get_edge_pixels(img, box):
     return jnp.concatenate(edge, axis=1)
 
 
-def adaptive_morphology(obs, center, min_size=11, delta_size=3, min_snr=20, min_corr=0.99, return_array=False):
+def make_bbox(obs, center_pix, min_size=11, delta_size=3, min_snr=20, min_corr=0.99):
     """
-    Create image of a Gaussian from the centered 2nd moments of the observation.
+    Make a bounding box for source at center.
```

Comment:
This function, which used to return the adaptive morphology, now only returns the box. I agree we should revert the change, but I wanted to double check with you if this was intended. I'm happy to make the change back.

Reply:
Yes, that is fine. I made the code more modular, so this function was broken out. No changes needed here.